lib/roo/xls/excel_2003_xml.rb

```diff
@@ -18,8 +18,10 @@ def initialize(filename, options = {})
       unless File.file?(@filename)
         fail IOError, "file #{@filename} does not exist"
       end
-      @doc = load_xml(@filename)
+      @doc = ::Roo::Utils.load_xml(@filename)
     end
+    namespace = @doc.namespaces.select{|xmlns, urn| urn == 'urn:schemas-microsoft-com:office:spreadsheet'}.keys.last
+    @namespace = namespace.nil? || namespace.empty? ? 'ss' : namespace.split(':').last
```

Comment:
On second thought, my suggestion is wrong. It would produce different, probably wrong, results than your code. Your code gets the last namespace key, mine gets the first.